### PR TITLE
[23.1] 23.1 Release Notes

### PR DESCRIPTION
**Pull request by dannon**
Standing release notes PR, everyone welcome to edit.

## How to test the changes?
(Select all options that apply)
- [ ] I've included appropriate [automated tests](https://docs.galaxyproject.org/en/latest/dev/writing_tests.html).
- [ ] This is a refactoring of components with existing test coverage.
- [ ] Instructions for manual testing are as follows:
  1. [add testing steps and prerequisites here if you didn't write automated tests covering all your changes]

## License
- [x] I agree to license these and all my past contributions to the core galaxy codebase under the [MIT license](https://opensource.org/licenses/MIT).

### CODE REVIEW

File: doc/source/releases/23.1_announce_user.rst [hunk 1]
```diff
@@ -0,0 +1,125 @@
...
+* Remove various fallback behaviors
+  (thanks to `@mvdbeek <https://github.com/mvdbeek>`__).
+  `Pull Request 16199`_
+
+Release Testing Team
+===========================================================
+
+A special thanks to the release testing team for testing many of the new features and reporting many bugs:
+
+<team members go here>
```
Comment (jdavcs) [inline doc/source/releases/23.1_announce_user.rst]:
```suggestion
* `Jennifer Hillman-Jackson <https://github.com/jennaj>`__
* `John Davis <https://github.com/jdavcs>`__
* `Luke Sargent <https://github.com/luke-c-sargent>`__
* `Martin Cech <https://github.com/martenson>`__
* `Michelle Savage <https://github.com/hujambo-dunia>`__
* `Stephen Mosher <https://github.com/slmosher>`__
```

File: doc/source/releases/23.1_announce_user.rst [hunk 2]
```diff
@@ -0,0 +1,125 @@
...
+  (thanks to `@SandyRogers <https://github.com/SandyRogers>`__).
+  `Pull Request 16179`_
+* Remove various fallback behaviors
+  (thanks to `@mvdbeek <https://github.com/mvdbeek>`__).
+  `Pull Request 16199`_
+
+Release Testing Team
+===========================================================
+
+A special thanks to the release testing team for testing many of the new features and reporting many bugs:
```
Comment (jdavcs) [inline doc/source/releases/23.1_announce_user.rst]:
```suggestion
A special thanks to the release testing team for testing many of the new features:
```
Sometimes there may be not too many bugs, and even if there are many, reporting them is only part of what the team does. So maybe just limit it to thanks for testing new features?

**Comment by hexylena** [comment]
@dannon when is the release officially scheduled? Just to make sure I get any user facing announcements in, or hand those off to someone else.

**Comment by dannon** [comment]
@hexylena It'd be great to wrap it up in the next week.  @natwhitaker offered to help with the user facing notes this time, I should have given you a heads up!  I'm sure y'all can work on them together if you want, though.

**Comment by hexylena** [comment]
Ahh that's, yeah good to know. Great to have help!

**Comment by natwhitaker** [comment]
Yes! Happy to help!

Natalie

From: Dannon ***@***.***>
Date: Monday, July 24, 2023 at 8:39 PM
To: galaxyproject/galaxy ***@***.***>
Cc: Natalie Whitaker ***@***.***>, Mention ***@***.***>
Subject: Re: [galaxyproject/galaxy] [23.1] 23.1 Release Notes (PR #16329)

      External Email - Use Caution





@hexylena<https://github.com/hexylena> It'd be great to wrap it up in the next week. @natwhitaker<https://github.com/natwhitaker> offered to help with the user facing notes this time, I should have given you a heads up! I'm sure y'all can work on them together if you want, though.

File: doc/source/releases/23.1_announce_user.rst [hunk 3]
```diff
@@ -0,0 +1,334 @@
...
+  `Pull Request 15656`_
+* Allow pending inputs in some collection operation tools
+  (thanks to `@mvdbeek <https://github.com/mvdbeek>`__).
+  `Pull Request 15892`_
+* Updates MGnify interactive tool to v1.1.0
+  (thanks to `@SandyRogers <https://github.com/SandyRogers>`__).
+  `Pull Request 16179`_
+* Remove various fallback behaviors
+  (thanks to `@mvdbeek <https://github.com/mvdbeek>`__).
+  `Pull Request 16199`_
```
Comment (nsoranzo) [inline doc/source/releases/23.1_announce_user.rst]: These are duplicated twice for some reason.

```suggestion
```

File: doc/source/releases/23.1.rst [hunk 4]
```diff
@@ -0,0 +1,2470 @@
...
+
+* Small test decorator improvements
+  (thanks to `@jmchilton <https://github.com/jmchilton>`__).
+  `Pull Request 16220`_
+* Small test decorator improvements
+  (thanks to `@jmchilton <https://github.com/jmchilton>`__).
+  `Pull Request 16220`_
+* Small test decorator improvements
+  (thanks to `@jmchilton <https://github.com/jmchilton>`__).
+  `Pull Request 16220`_
```
Comment (nsoranzo) [inline doc/source/releases/23.1.rst]:
```suggestion
```

File: doc/source/releases/23.1.rst [hunk 5]
```diff
@@ -0,0 +1,2470 @@
...
+  `Pull Request 15972`_
+* Code cleanups from ruff and pyupgrade
+  (thanks to `@nsoranzo <https://github.com/nsoranzo>`__).
+  `Pull Request 16035`_
+* Dataset chunking tests (and small fixes)
+  (thanks to `@jmchilton <https://github.com/jmchilton>`__).
+  `Pull Request 16069`_
+* Add hdf4 datatype
+  (thanks to `@TheoMathurin <https://github.com/TheoMathurin>`__).
+  `Pull Request 16105`_
```
Comment (nsoranzo) [inline doc/source/releases/23.1.rst]:
```suggestion
```

File: doc/source/releases/23.1.rst [hunk 6]
```diff
@@ -0,0 +1,2470 @@
...
+  `Pull Request 15892`_
+* Updates MGnify interactive tool to v1.1.0
+  (thanks to `@SandyRogers <https://github.com/SandyRogers>`__).
+  `Pull Request 16179`_
+* Allow pending inputs in some collection operation tools
+  (thanks to `@mvdbeek <https://github.com/mvdbeek>`__).
+  `Pull Request 15892`_
+* Updates MGnify interactive tool to v1.1.0
+  (thanks to `@SandyRogers <https://github.com/SandyRogers>`__).
+  `Pull Request 16179`_
```
Comment (nsoranzo) [inline doc/source/releases/23.1.rst]:
```suggestion
```

File: doc/source/releases/23.1.rst [hunk 7]
```diff
@@ -0,0 +1,2470 @@
...
+
+* e2e test for workflow license selector
+  (thanks to `@jmchilton <https://github.com/jmchilton>`__).
+  `Pull Request 16243`_
+* e2e test for workflow license selector
+  (thanks to `@jmchilton <https://github.com/jmchilton>`__).
+  `Pull Request 16243`_
+* e2e test for workflow license selector
+  (thanks to `@jmchilton <https://github.com/jmchilton>`__).
+  `Pull Request 16243`_
```
Comment (nsoranzo) [inline doc/source/releases/23.1.rst]:
```suggestion
```

File: doc/source/releases/23.1.rst [hunk 8]
```diff
@@ -0,0 +1,2470 @@
...
+
+* Record input datasets and collections at full parameter path
+  (thanks to `@mvdbeek <https://github.com/mvdbeek>`__).
+  `Pull Request 15978`_
+* Record input datasets and collections at full parameter path
+  (thanks to `@mvdbeek <https://github.com/mvdbeek>`__).
+  `Pull Request 15978`_
+* Record input datasets and collections at full parameter path
+  (thanks to `@mvdbeek <https://github.com/mvdbeek>`__).
+  `Pull Request 15978`_
```
Comment (nsoranzo) [inline doc/source/releases/23.1.rst]:
```suggestion
```

File: doc/source/releases/23.1.rst [hunk 9]
```diff
@@ -0,0 +1,2470 @@
...
+  `Pull Request 16096`_
+* read job_conf directly from `config_dir` instead of computing it again from `config_file`
+  (thanks to `@bernt-matthias <https://github.com/bernt-matthias>`__).
+  `Pull Request 15596`_
+* Change confusing pulsar logs message
+  (thanks to `@kysrpex <https://github.com/kysrpex>`__).
+  `Pull Request 16038`_
+* Fix job failure handling when condor indicates job failure
+  (thanks to `@mvdbeek <https://github.com/mvdbeek>`__).
+  `Pull Request 16096`_
```
Comment (nsoranzo) [inline doc/source/releases/23.1.rst]:
```suggestion
```

**Review by nsoranzo — COMMENTED** [review]
Something is broken with the script generating this file.

**Comment by dannon** [comment]
> Something is broken with the script generating this file.

Yep, I noticed -- in the commit history here I deleted those blocks completely and let them get regenerated once to fix it up; haven't had a chance to dig into the error yet though.

**Comment by nsoranzo** [comment]
@dannon Can you please cherry-pick commit 6d358124174ec686d54df09f9eb5b1c39c654a39 which got lost in the last rebase? Thanks.

**Comment by dannon** [comment]
@nsoranzo Of course, sorry about that.  I fixed the bug in galaxy-release-util that was causing the duplicates and regenerated this, but forgot to pull that down before rebasing.

File: doc/source/releases/23.1_announce_user.rst [hunk 10]
```diff
@@ -0,0 +1,184 @@
...
+Stay on top of your carbon emissions contributions by enabling our new reporting
+feature! Initial implementation of carbon emissions reporting has now been added
+to the job information page. Monitor your carbon footprint in real-time and
+become inspired to better your relationship with the environment! Currently,
+this feature assumes global average values for carbon intensity and power usage
+effectiveness.
+
+History Archival Feature!
+-------------------------
+Your history menu now has a new option move a history out of your active history
```
Comment (nsoranzo) [inline doc/source/releases/23.1_announce_user.rst]:
```suggestion
Your history menu now has a new option to move a history out of your active histories
```

File: doc/source/releases/23.1_announce_user.rst [hunk 11]
```diff
@@ -0,0 +1,184 @@
...
+and into an archive! When archiving a history, you will be presented with two
+options: one to “Keep the Storage Space” and one to “Free the Storage Space”
```
Comment (nsoranzo) [inline doc/source/releases/23.1_announce_user.rst]:
```suggestion
options: either to “Keep the Storage Space” or to “Free the Storage Space”
```

File: doc/source/releases/23.1_announce_user.rst [hunk 12]
```diff
@@ -0,0 +1,184 @@
...
+taken up by the history. Keeping the storage space will keep the contents of the
+history where they are, allowing them to be restored at any moment, but will
+prevent the UI from allowing certain operations on the history. Freeing the
+storage space will prompt you to package and export the history to a permanent
+remote source as a backup snapshot, and then purge the history and its contents
+to free up storage space. There will be a simple list, titles Archived
```
Comment (nsoranzo) [inline doc/source/releases/23.1_announce_user.rst]:
```suggestion
to free up storage space. There will be a simple list, titled Archived
```

File: doc/source/releases/23.1_announce_user.rst [hunk 13]
```diff
@@ -0,0 +1,184 @@
...
+Histories, for you to explore your archived histories and restore and reimport
+them, as needed!
+
+Galaxy Notification System!
+---------------------------
+The previous Galaxy notification system has been overhauled to support more
+complex scenarios and features! As users, you will now have more control of your
+notifications, including the option to subscribe/unsubscribe from certain times
```
Comment (nsoranzo) [inline doc/source/releases/23.1_announce_user.rst]:
```suggestion
notifications, including the option to subscribe/unsubscribe from certain types
```

I guess?

**Review by guerler — APPROVED** [review]

**Pull request merged**